test/statefulset_storage_test.go

```diff
@@ -0,0 +1,490 @@
+package test
+
+import (
+	"strconv"
+	"time"
+
+	goctx "context"
+	asdbv1beta1 "github.com/aerospike/aerospike-kubernetes-operator/api/v1beta1"
+	"github.com/aerospike/aerospike-kubernetes-operator/pkg/utils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe(
+	"STSStorage", func() {
+		ctx := goctx.TODO()
+
+		Context(
+			"When doing valid operations", func() {
+
+				BeforeEach(
+					func() {
+						clusterName := "sts-storage"
```

Comment:
Make it global, at the context level. No need to repeat in every `IT`
`clusterName := "sts-storage"
						clusterNamespacedName := getClusterNamespacedName(
							clusterName, namespace,
						)`